bpf/failsafes/failsafes.go

```diff
@@ -95,9 +97,33 @@ func (m *Manager) ResyncFailsafes() error {
 			log.WithField("proto", p.Protocol).Warn("Ignoring failsafe port; protocol not supported in BPF mode.")
 			return
 		}
-		k := MakeKey(ipProto, p.Port, outbound)
+
+		// Slice up the CIDR and convert the string representations to uints
+		parts := strings.Split(p.Net, "/")
```

Comment:
Think we have a utility function for parsing CIDRs or IPs.  `net.ParseCIDROrIP`  in libcalico-go?

Reply:
Ah yes, I'll remove this logic since it appears to cause FV issues too and go with something a bit safer.